be/src/io/fs/local_file_system.cpp

```diff
@@ -182,7 +183,11 @@ Status LocalFileSystem::directory_size(const Path& dir_path, size_t* dir_size) {
     if (std::filesystem::exists(dir_path) && std::filesystem::is_directory(dir_path)) {
         for (const auto& entry : std::filesystem::recursive_directory_iterator(dir_path)) {
             if (std::filesystem::is_regular_file(entry)) {
-                *dir_size += std::filesystem::file_size(entry);
+                try {
+                    *dir_size += std::filesystem::file_size(entry);
+                } catch (const std::exception& e) {
+                    LOG(INFO) << "{}", e.what();
```

Comment:
Print more context in the log e.g. 
```suggestion
                    LOG(INFO) << "failed to get file size, err: {}", e.what();
```